oioioi/teachers/tests.py

```diff
@@ -7,6 +7,9 @@
 from oioioi.contests.tests.utils import make_user_contest_admin
 from oioioi.teachers.models import Teacher
 
+from oioioi.teachers.utils import add_user_to_contest_as as autc, \
```

Comment:
it would be more readable if we use full name add_user_to_contest_as not shortcut autc.